<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page2</class>
 <widget class="QMainWindow" name="page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>221</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="file2input">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>文件2：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>文件1：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="file1input">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="file1but">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>50</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>选择文件1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="file2but">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>120</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>选择文件2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startcup">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>180</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始计算</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/png/img/1.png</normaloff>:/png/img/1.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_PSI">
    <property name="title">
     <string>本地PSI</string>
    </property>
   </widget>
   <addaction name="menu_PSI"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
